web/pdf_viewer.js

```diff
@@ -247,6 +251,9 @@ class PDFViewer {
       );
     }
     this.container = options.container;
+    this.#updateContainerHeightCss(this.container.clientHeight);
```

Comment:
If the nit below is addressed, this could be simplified to just:
```suggestion
    this.#updateContainerHeightCss();
```